app/assets/stylesheets/application.scss

```diff
@@ -1,4 +1,5 @@
 @import "bootstrap-sprockets";
 @import "theme/variables";
 @import "bootstrap";
-@import "theme/bootswatch";
\ No newline at end of file
+@import "theme/bootswatch";
+@import "summary-card";
```

Comment:
Might be good to start putting together a directory structure too. Within the stylesheets, we could have a set of folders like:

```
/base/
/components/
/helpers/
/pages/
```

This would be an ideal candidate to include in the `/components` sub-folder for stylesheets.